tests/lib/eslint.js

```diff
@@ -1893,6 +1894,22 @@ describe("eslint", function() {
             messages = eslint.verify(codeB, config, filename, false);
             assert.equal(messages.length, 1);
         });
+
+
+        it("should report the docURI provided by plugin", function() {
+            var config = {
+                rules: {"test-plugin/test-rule": 2},
+                getDocURI: function(ruleName) {
+                    return "http://path.to/docs/" + ruleName;
+                }
+            };
+            var code = "var a = \"trigger violation\";";
+
+            eslint.reset();
+            var messages = eslint.verify(code, config, filename, false);
+
+            assert.equal(messages[0].docURI, "http://path.to/docs/test-plugin/test-rule");
+        });
```

Comment:
I think it should be in a different `describe` since this one is about code with inline comments that modify the configuration.
Good idea about adding more comments to test old and new style as well, i'm 👍 for that!